src/test/java/com/tagtraum/perf/gcviewer/imp/TestDataReaderUJLG1JDK11.java

```diff
@@ -180,4 +180,32 @@ public void testPauseYoungConcurrentStartMetadataGcThreshold() throws Exception
         assertThat("event type", model.get(0).getExtendedType().getType(), is(Type.UJL_PAUSE_YOUNG));
         assertThat("total heap", model.get(0).getTotal(), is(256 * 1024));
     }
+
+    @Test
+    public void testPauseNoPromotion() throws IOException {
+        TestLogHandler handler = new TestLogHandler();
+        handler.setLevel(Level.WARNING);
+        GCResource gcResource = new GcResourceFile("byteArray");
+        gcResource.getLogger().addHandler(handler);
+        InputStream in = new ByteArrayInputStream(
+                ("[12.029s][info][gc,start      ] GC(9) Pause Young (Normal) (G1 Evacuation Pause)\n" +
+                        "[12.029s][info][gc,task       ] GC(9) Using 8 workers of 8 for evacuation\n" +
+                        "[12.034s][info][gc,phases     ] GC(9)   Pre Evacuate Collection Set: 0.0ms\n" +
+                        "[12.034s][info][gc,phases     ] GC(9)   Evacuate Collection Set: 4.4ms\n" +
+                        "[12.034s][info][gc,phases     ] GC(9)   Post Evacuate Collection Set: 0.5ms\n" +
+                        "[12.034s][info][gc,phases     ] GC(9)   Other: 0.2ms\n" +
+                        "[12.034s][info][gc,heap       ] GC(9) Eden regions: 143->0(142)\n" +
```

Comment:
I think your test data is not correct. At least the parser does not read it correctly. If you add following lines top your test you will see that pre used of young is 0 but should be 143M.
```
assertThat("pre", model.get(0).getPreUsed(), is(162*1024)); // pass
assertThat("pre", model.get(0).getPostUsed(), is(19*1024)); // pass

AbstractGCEvent<?> event = model.get(0).getPhases().get(0);
assertThat("young pre", event.getGeneration(), is(AbstractGCEvent.Generation.YOUNG)); // pass
assertThat("young pre", event.getPreUsed(), is(143*1024)); // fail
assertThat("young pre", event.getPostUsed(), is(0L)); // pass
```
Your example would not produce the desired error if parsed correctly:
```
int young = 143 - 0; // 143
int all = 162 - 19; // 143
int promotion = young - all; // 0
```
The format in this test differs from the format used in your issue #235 that produces the error in "real life".